userspace/openpilot_python_dependencies.sh

```diff
@@ -2,20 +2,19 @@
 
 echo "Installing python for openpilot"
 
-# Install pyenv
-export PYENV_ROOT="/usr/local/pyenv"
-curl https://pyenv.run | bash
-export PATH="$PYENV_ROOT/bin:$PATH"
-eval "$(pyenv init -)"
+if ! command -v "uv" > /dev/null 2>&1; then
```

Comment:
it won't be installed

Reply:
I removed the if condition check, if needed the if condition is capable to check whether uv file exists.